src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -229,10 +229,12 @@ std::vector<CPUSpecificParams> filterSpecificParams() {
     return specificParams;
 }
 
-std::vector<CPUSpecificParams> filterSpecificParams_Brgemm() {
+std::vector<CPUSpecificParams> filterSpecificParams_Brgemm(bool tryBrgAVX2 = false) {
```

Comment:
Why cannot we add brgemm_avx2 unconditionally? My expections is that brgemm coverage between avx512 and avx2 should be identical

Reply:
brgemm_matmul_t can't support avx2 in stock onednn. That is why the reason we have to pass on argument into to decide whether to add the brgemm_avx2 into list.

Reply:
Got it, thanks. Please add this info as comment here.

Reply:
Applied